src/styles/main.scss

```diff
@@ -1,3 +1,203 @@
+}
+
+.header {
+  width: 100%;
+  display: flex;
+  padding: 0 50px;
+  align-items: center;
+  justify-content: space-between;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  display: block;
+}
+
+.nav {
+
+  &__list {
+    display: flex;
+    list-style: none;
+  }
+
+  &__item:not(:last-child) {
+    margin-right: 20px;
+  }
+
+  &__link {
+    display: block;
+    text-decoration: none;
+    text-transform: uppercase;
+    color: #060b35;
+    padding: 23px 0;
+  }
+
+}
+
+.nav__link:hover {
+  color: $blue-color;
+}
+
+.is-active {
+  position: relative;
+  color: $blue-color;
+}
+
+.is-active::after {
+  content: "";
+  position: absolute;
+  width: 37px;
+  height: 4px;
+  background: $blue-color;
+  border-radius: 8px;
+  bottom: 0;
+  left: 0;
+  right: 0;
+}
+
+// body,
+// p {
+//   margin: 0;
+// }
+
+.cards {
+  display: flex;
+  flex-wrap: wrap;
+  max-width: 950px;
+  justify-content: center;
+  padding: 50px 0;
+}
+
+.home-page__cards {
+  margin: 0 auto;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &:not(:nth-child(4n)) {
+    margin-right: 48px;
+  }
+
+  &:nth-last-child(-n + 4) {
+    margin-top: 46px;
+  }
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    margin-top: 32px;
+  }
+
+  &__title {
+    font-size: 12px;
+    margin: 40px 16px 0;
+    line-height: 18px;
+    font-weight: 500;
+  }
+
+  &__product-code {
+    display: block;
+    font-size: 10px;
+    font-weight: 400;
+    color: $secondary-color;
+    align-self: start;
+    line-height: 14px;
+    margin-left: 16px;
+    margin-top: 4px;
+  }
+
+  &__price {
+    display: flex;
+    margin-top: 24px;
+
+    &-text {
+      font-size: 12px;
+      font-weight: 400;
+      color: $secondary-color;
+      margin-right: 85px;
+      align-self: center;
+    }
+
+    &-count {
+      font-size: 16px;
+      text-align: center;
+      line-height: 18px;
+      font-weight: 700;
+      color: #060b35;
+    }
+  }
+
+  &__buy-btn {
+    color: $white-color;
+    background-color: $blue-color;
+    text-decoration: none;
+    width: 166px;
+    height: 40px;
+    border-radius: 5px;
+    margin: 16px 0;
+    text-align: center;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 40px;
+    border: 1px solid $blue-color;
+
+    &:hover {
+      color: $blue-color;
+      background-color: $white-color;
+      border: 1px solid $blue-color;
+    }
+  }
+}
+
+.container {
+  display: flex;
+  margin-top: 16px;
+  align-items: center;
+
+  &__reviews {
+    text-align: center;
+    line-height: 16px;
+    font-size: 10px;
+    font-weight: 400;
+    margin-left: 22.5px;
+  }
+}
+
+.stars {
+  display: flex;
+  height: 16px;
+
+  &__star {
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    width: 16px;
+    height: 16px;
+
+    &:nth-child(-n+4) {
```

Comment:
it should be styles for stars--4, and other amount modifiers
 make scss loop for it
![Screenshot from 2023-04-16 16-00-37](https://user-images.githubusercontent.com/33029864/232313329-5852cf20-4a8c-4c87-9f1c-8c85af8c2daa.png)